### PR TITLE
feat: 💄 Divider support `plain` for lighter style

### PR DESCRIPTION
**Pull request by afc163**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / document update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

image

增加一个新的正文风格的分割文字风格，标题样式比较重，大多数场景都需要修改字体样式。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Divider add `plain` prop which allows a non-heading style divider text. |
| 🇨🇳 Chinese | Divider 新增 `plain` 属性，可用于设置一个非标题样式的分割文字。  |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/divider/demo/customize-style.md](https://github.com/ant-design/ant-design/blob/feat-text-divider/components/divider/demo/customize-style.md)
[View rendered components/divider/demo/horizontal.md](https://github.com/ant-design/ant-design/blob/feat-text-divider/components/divider/demo/horizontal.md)
[View rendered components/divider/demo/no-heading.md](https://github.com/ant-design/ant-design/blob/feat-text-divider/components/divider/demo/no-heading.md)
[View rendered components/divider/demo/vertical.md](https://github.com/ant-design/ant-design/blob/feat-text-divider/components/divider/demo/vertical.md)
[View rendered components/divider/demo/with-text.md](https://github.com/ant-design/ant-design/blob/feat-text-divider/components/divider/demo/with-text.md)

### CODE REVIEW

**Comment by zombieJ** [comment]
`asHeading` 这个不太符合现在的 api 风格，直接 `heading` or `type='heading'` 如何？

**Comment by afc163** [comment]
type 已经占用了用不了，heading 不明确。还有别的名字没。

**Comment by afc163** [comment]
改成 `plain` 了。

File: components/divider/style/index.less [hunk 1]
```diff
@@ -107,4 +92,10 @@
   &-vertical&-dashed {
     border-width: 0 0 0 1px;
   }
+
+  &-heading&-with-text {
```
Comment (zombieJ) [inline components/divider/style/index.less:96]: `-heading` 是不是也要改改

**Review by zombieJ — APPROVED** [review]

**Pull request merged**